### ISSUE TITLE
Clicking custom extension in Extensions window crashes Revit

### ISSUE THREAD

**Issue by Tyree** [eirannejad/pyRevit#734]
pyRevit 4.6.23 on Revit 2019.2
I am working on creating a custom extension. Everything seemed to be working fine. The tab is showing up on the ribbon, tools work, etc. 

But, I went to look at the list of extensions and saw the custom one was not set as enabled or disabled. When I clicked on it, I got this error and it'll sometimes even crash Revit completely when I click it.

Contents of my extension.json:
```
{
    "builtin": "False",
    "default_enabled": "False",
    "type": "extension",
    "rocket_mode_compatible": "False",
    "name": "APDynamo",
    "description": "AP Custom Dynamo Scripts",
    "author": "Matt Tyree",
    "author_profile": "",
    "url": "",
    "website": "",
    "image": "",
    "templates": {
        "author": "",
        "docpath": ""
    },
    "dependencies": []
}
```
![error](https://user-images.githubusercontent.com/5395363/67796924-3b917d00-fa57-11e9-8ffe-888aeae99772.PNG)

**Comment by eirannejad** [comment]
There was a bug that got fixed in 4.7. Fill in the `author_profile` and `url` and I think it would bypass the bug.

**Comment by eirannejad** [comment]
Resolved in 4.7
